deploy/docker/get_buttonmen_site_ipaddr

```diff
@@ -76,10 +76,6 @@ def connect_boto_clients():
     'ecs': boto3.client('ecs'),
   }
 
-def docker_reponame(git_info):
-  prefix = (git_info['site_type'] == 'replay') and 'replay' or 'buttonmen'
-  return f"{prefix}-{git_info['reponame']}/{git_info['branch']}"
-
```

Comment:
I could have updated this function, but it turns out it's not used by this script, so i removed it instead.

And, yes, these things should live in a library and not be synced by hand, but that's not this minute's problem.